_posts/2024-02-19-jco-1.0.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Announcing Jco 1.0"
+author: "Yosh Wuyts"
+date: "2024-02-21"
+github_name: "yoshuawuyts"
+excerpt_separator: <!--end_excerpt-->
+---
+
+We’re happy to announce the 1.0 release of [jco]: a native Javascript
+WebAssembly runtime and toolchain built for WebAssembly Components. Jco can
```

Comment:
> I'd feel much more comfortable if we could call it an `experimental transpilation-based runtime` but I guess that doesn't exactly roll off the tongue.

Yeah; that also leads to questions such as: "if Jco is experimental, why is it at 1.0 already?" We know that 1.0 means API-stable, not necessarily battle-tested - but that's probably too nuanced to communicate in an announcement post.

I think leaving off the "experimental" prefix, and being careful about the claims we're making should be enough to let people interested in this evaluate for themselves whether it meets their expectations. I suspect for a lot of folks it will already!